manifests/admin.pp

```diff
@@ -0,0 +1,38 @@
+# @summary Run a django-admin command
+#
+# @param command
+#   The command to run
+#
+# @param refreshonly
+#   The command should only be run as a refresh mechanism for when a dependent
+#   object is changed.
+#
+# @param unless
+#   A test command that checks the state of the target system and restricts
+#   when the exec can run.
+#
+# @param path
+#   The path to look for commands.
+#
+# @param pulp_settings
+#   The pulp settings file to use
+#
+# @see exec
+define pulpcore::admin(
+  String $command = $title,
+  Boolean $refreshonly = false,
+  Optional[String] $unless = undef,
+  Array[Stdlib::Absolutepath] $path = ['/usr/bin'],
+  Stdlib::Absolutepath $pulp_settings = $pulpcore::settings_file,
+) {
+  File <| title == $pulp_settings |>
+  -> exec { "django-admin ${title}":
```

Comment:
So once @evgeni has modified the RPM design so that django 3's `django-admin` does not conflict with django 2, we can update the `exec` statement in a single place and have it be correct for the whole module.

I really like this approach. LGTM.